pop.css

```diff
@@ -0,0 +1,151 @@
+.pop-up-background {
+  height: 100vh;
+  width: 100vw;
+  position: fixed;
+  top: 0;
+  left: 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  backdrop-filter: blur(10px);
+}
+
+.pop-up-card {
+  border: 1px solid #dfe1e6;
+  border-radius: 4.26vw;
+  background-color: #fff;
+  width: 91.46vw;
+  overflow: auto;
+}
+
```

Comment:
hey @pulgosodog not that you popup should be responsive and should not only be limited on 1440 screens

https://github.com/pulgosodog/Portfolio-template-1/assets/65068771/f80c7079-00c4-4b93-a2a4-5aee332ac871


https://github.com/pulgosodog/Portfolio-template-1/assets/65068771/e6340974-ddd6-429a-b8d1-866d9cb04cb6